src/server/db_slice.cc

```diff
@@ -219,9 +220,13 @@ unsigned PrimeEvictionPolicy::Evict(const PrimeTable::HotspotBuckets& eb, PrimeT
       return 0;
     }
 
+    std::string tmp;
+    std::string_view key = last_slot_it->first.GetSlice(&tmp);
+
     DbTable* table = db_slice_->GetDBTable(cntx_.db_index);
     PerformDeletion(last_slot_it, db_slice_->shard_owner(), table);
     ++evicted_;
+    db_slice_->SendInvalidationTrackingMessage(key);
```

Comment:
In the general case it's not safe to hold a key string_view after it was deleted, you can use GetString()

Reply:
sounds good, actually I changed to use `ToString()` which uses `GetString()` inside...